### PR TITLE
Fix issue with race conditions and undefined issues

### PR DESCRIPTION
**Pull request by nimitzshah**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "posthtml-content",
   "description": "Flexible content transform for posthtml",
-  "version": "0.0.1",
+  "version": "0.0.2",
```
Comment (jescalan) [inline package.json]: Could you keep the version the same? I have a process for bumping it

**Comment by jescalan** [comment]
Also looks like you have some lint errors...

**Comment by nimitzshah** [comment]
Doh! Ill fix that up

**Comment by nimitzshah** [comment]
Fixed!

**Pull request merged**